Source/WebCore/Modules/identity/IdentityCredentialsContainer.cpp

```diff
@@ -48,6 +50,22 @@ void IdentityCredentialsContainer::get(CredentialRequestOptions&& options, Crede
     if (!performCommonChecks(options, promise))
         return;
 
+    if (!m_document->hasFocus()) {
```

Comment:
Nit: we should store `m_document` in a local `RefPtr` then use that throughout this function instead of accessing `m_document` directly, e.g.:
```
RefPtr document = m_document.get();
if (!document->hasFocus()) {
```